src/software/ai/hl/stp/play/corner_kick_play.cpp

```diff
@@ -157,6 +157,11 @@ void CornerKickPlay::getNextTactics(TacticCoroutine::push_type &yield, const Wor
 
     LOG(DEBUG) << "Finished aligning to ball";
 
+    // Restart the align to ball tactic, but now loop forever so that
+    // the robot holds position until it's ready to kick
+    align_to_ball_tactic =
+         std::make_shared<MoveTactic>(true);
+
```

Comment:
why is this necessary?

Reply:
Thanks for pointing this out, I think this was just a leftover from me messing around with stuff. It's not required, so removed.

Reply:
Whoops, it's not, just a leftover from testing. Removed and looks like the simulated test is still fine.